added & tested my files

abhilashreddyy opened the pull request:
Name  : Yammanuru Abhilash Reddy (row number 123 in excel file)(Task: 961-966)
ASUID : 1222240799

Successfully added and tested task files. Created tasks from 961-966.  Please review

swarooprm commented:
Update: Reviews discussed in detail during the office hours.

abhilashreddyy commented:
Hey @swarooprm. 
1. I have made the suggested changes. 
2. Read and corrected datasets thoroughly for any grammatical, spelling mistakes, or inconsistencies.
3. Added 2 more datasets.
4. Converted the Language Translation Tasks data from Unicode to a readable format.

swarooprm reviewed (commented):
- In the relation extraction task, you need to mention in definition what are the possible options for relations.
- In the question generation task, if you have multiple questions per input, maker sure to represent those in the form of a list in case of output of instances. Let output of positive examples be only one question.
Others look good.

swarooprm commented:
@abhilashreddyy, can you address reviewer comments so that I can evaluate your HW?

swarooprm commented:
> @abhilashreddyy, can you address reviewer comments so that I can evaluate your HW?

A gentle reminder! @abhilashreddyy

abhilashreddyy commented:
1. I have added categories to the relation generation task and converted it to a classification task
2. There is only one question per instance in the question generation task and the output of the question generation instance us in list format
NOTE : Apologies for the delayed commit, I was using this git repository on an old email address which is heavily spammed. Hence I lost the suggestion messages. I would be really grateful if can still consider my assignment

abhilashreddyy commented:
Yes, I would like to contribute these tasks to the research. Will submit the pull request tomorrow after the NLP test.
